src/remoteConnector.ts

```diff
@@ -1028,6 +1030,71 @@ export default class RemoteConnector extends Disposable {
 		}
 	}
 
+	private async configureMachineSettings(wsConfig: WorkspaceConfig) {
```

Comment:
Should not this logic implemented in remote extension? It has access to machine setitngs, supervisor and VS Code api. I don't see a point to implement in local extension.